staging/src/k8s.io/apiserver/pkg/registry/generic/registry/dryrun.go

```diff
@@ -69,28 +69,28 @@ func (s *DryRunnableStorage) GetList(ctx context.Context, key string, opts stora
 }
 
 func (s *DryRunnableStorage) GuaranteedUpdate(
-	ctx context.Context, key string, ptrToType runtime.Object, ignoreNotFound bool,
+	ctx context.Context, key string, out runtime.Object, ignoreNotFound bool,
```

Comment:
This is like, memory to store the object in that has the right type. I don't think "out" is a good name for that.

Reply:
I thought `out` would be fine as that's where the actual output of operations executed in `Get()` and `GuaranteedUpdate()` gets stored. What would you rather we change it to?

Reply:
It's a hard question!

The reason this is an input parameter is because this function needs to know the type to use, there's no way for it to construct a variable of the right type. That we also use this to store the result is accidental -- if that were the primary purpose here, we'd return that as an output parameter.

The best I can come up with at the moment is `resultDestination` or maybe just `destination` but I'm not in love with that, either.

Reply:
"memory to store the object" sounds a lot like output!
And yes, this is also being used to input the type.
I think both `resultDestination` and `destination` are fine and sufficient improvements to `ptrToType`.